core/commands/config.go

```diff
@@ -41,15 +41,15 @@ repository.`,
 much like 'git config'. The configuration values are stored in a config
 file inside your IPFS repository.
 
-EXAMPLES:
+Examples:
 
 Get the value of the 'datastore.path' key:
 
-  ipfs config datastore.path
+  $ ipfs config datastore.path
```

Comment:
That's true, would backticks clarify it better? Or they could have been accidentally copied as well.

Though `$` is used more often: https://github.com/git/git/blob/6ccc71a9d02726d59c1de818febf85a836f4e418/contrib/subtree/git-subtree.txt#L250, so it should be fine.